code/src/index.css

```diff
@@ -1,13 +1,88 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(255, 144, 162);
+  font-family: 'Roboto', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.card {
+  margin: 20px;
+  box-shadow: 10px 10px 4px -4px black;
+  border: 2px solid black;  
+  padding: 0 20px;
+  background-color: white;
 }
+
+/*** MessageInput ***/
+
+textarea, input {
+  display: block;
+  margin-bottom: 20px;
+}
+
+.text-input {
+  padding: 5px;
+  width: calc(100% - 10px);
+  height: 50px;
+  background-color: rgb(243, 243, 243);
+  font-size: 12px;
+}
+
+.submit-button {
+  border: 2px solid black;
+  border-radius: 20px;
+  padding: 10px 20px;  
+  background-color: white;
+  font-size: 12px;
+}
+
+/*** Message ***/
+
+.card-footer {
+  display: flex;
+  justify-content: space-between;
+  color: grey;
+  font-size: 12px;
+}
+
+.likes-container {
+  display: flex;
+}
+
+button {
+  margin: 0;
+  border: none;
+  padding: 0;
+  background-color: white;
+}
+
+img {
+  display: flex;
+  height: 20px;
+}
+
+.likes {
+  display: inline-block;
+  margin-left: 10px;
+}
+
+/*** Media queries ***/
```

Comment:
For this specific project, I would almost recommend removing the media queries and instead adding a max-width and fixed margin to the card,  it makes the transitions between different screensizes smoother! The padding on the body I think looks great on all screensizes 😊